### DISCUSSION TITLE
Quick question about the "bot" folder in the tools folder

### DISCUSSION THREAD

**Discussion by silverraa** [Cvolton/GMDprivateServer#749] [Q&A]
Does someone created a GDPS bot with those files in the "bot" folder? If you do, how did you make it?

Since i'm planning for a GDPS bot that will notify us if there's a new level rated, and also for Trial Mods that were gonna send levels.

**Comment by Cvolton** [comment]
an implementation that makes use of the current features there is available here
https://github.com/Cvolton/pythonCvoltonBot

**Reply by silverraa** [reply]
Thanks, but can i transfer the contents on the "bot" folder to the CvoltonBot folder?

**Reply by ghost** [reply]
I'm pretty sure you can't because the GDPS is PHP and CvoltonBot is Python but it might work, just make a backup of the "bot" folder before trying
